docs/configuration/index.md

```diff
@@ -374,21 +374,48 @@ repository.  Defaults to 4, that is, patches will look like `PatchNNNN:
 minimum width".
 
 #### notifications
-
-There is only one notification configuration you can set up right now: enable
-the "Congratulations!" comment which will packit send after a successful build of a
-pull request is done.
-
-The default behaviour is not to send the comment with instructions how to install a
-package with the change implemented in the pull request:
+##### pull_request
+###### successful_build
+(*bool*)
+Enable "Congratulations!" comment which will be posted by Packit after a successful build of a
```

Comment:
I'm not sure if people still understand the "Congratulations!" since most of the current users can't remember this being the default behaviour. But since it's quite well described later on, we can leave it there.